README.md

```diff
@@ -474,7 +472,7 @@ run this instead:
 sudo npm install -g @openwhisk/wskdebug --unsafe-perm=true --allow-root
 ```
 
-The dependency `ngrok` requires full write permission in `/usr/local/lib/node_modules` during its custom install phase. This is a [known ngrok issue](https://github.com/inconshreveable/ngrok/issues/429).
+The dependency `ngrok` requires full write permission in `/usr/local/lib/node_modules` during its custom install phase. This is a [known ngrok issue](https://github.com/bubenshchykov/ngrok/issues/87).
```

Comment:
Wondering if the `sudo npm install...` command just above this block should be changed to just installing ngrok instead of openwhisk/debug?

Reply:
This is not the normal install command. This is the troubleshooting section which I kept in for folks still having an older version...

This PR was all about making the normal command easier from the very unusual

```
npm install -g @openwhisk/wskdebug --unsafe-perm=true
```

to the usual

```
npm install -g @openwhisk/wskdebug
```

Reply:
right.  guess it makes sense to leave comment in for a little while (or until the next release).  without the history, it just reads a little oddly since it is telling people about problems with an install mode that isn't mentioned anywhere else in the doc

Reply:
Yes, will remove after the next release.